### DISCUSSION TITLE
Can I get the latest available version number via the API?

### DISCUSSION THREAD

**Discussion by PEZ** [microsoft/vscode-discussions#228] [Extension Development QnA] [answered]
It is common that users of my extension do not realize they are running on an old version. Im not making the extension inform about the version used when it starts:

``` typescript
  chan.appendLine(
    'Calva version used: v' +
      vscode.extensions.getExtension('betterthantomorrow.calva').packageJSON.version
  );
```

I'd like to include info about the latest available version. Can that be done?

This is somewhat related to:

* #126

**Comment by isidorn** [comment, marked as answer]
There is no API to get the version of the latest available version on the Marketplace.
There is only the API which you found for the local version installed.
